pkgs/tools/misc/mongodb-compass/default.nix

```diff
@@ -82,10 +82,12 @@ let
     if stdenv.hostPlatform.system == "x86_64-linux" then
       fetchurl {
         url = "https://downloads.mongodb.com/compass/mongodb-compass_${version}_amd64.deb";
-        sha256 = "sha256-ceQp4EiLEWy8niGC0uUdWJrvmdt9Ijt29YdLt7vtcyY=";
+        sha256 = "sha256-w66ky83VkPhoa4OibzcsaLSWhoI4DQvUGWaKsBuCB7Q=";
       }
     else
       throw "MongoDB compass is not supported on ${stdenv.hostPlatform.system}";
+      # NOTE While MongoDB Compass is available to darwin, I do not have resources to test it
+      # Feel free to make a PR adding support if desired
```

Comment:
We don't need to add such comments but now that you added it we can also keep it 🙃